src/tail/session.rs

```diff
@@ -32,16 +33,36 @@ impl Session {
         metrics_port: u16,
         verbose: bool,
     ) -> Result<(), failure::Error> {
+        let mut session = Session { tail_id: None };
```

Comment:
why no tail id

Reply:
it isn't populated yet at this point; in `start`, we call the api to initiate the session, then populate this with the response. should i leave an inline comment?

Reply:
i'd just make it `Session::new()` tbh